File: java/Java programming 2/src/session14_gennerrics/entity/Product.java
package session14_gennerrics.entity;

import Slot_4.BaiTapCuaHang.model.StoreImpl;

public class Product extends Entity<Integer>{
    private String name;
    private String description;
    private double price;

    public Product(Integer id) {
        super(id);
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

}

File: C#/.Net/ProDB/model/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProDB.model
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal price {get; set;}
        public string description{get;set;}

    }
}